antarest/common/interfaces/eventbus.py

```diff
@@ -0,0 +1,67 @@
+from abc import ABC, abstractmethod
+from typing import Any, Callable, Optional, List, TypeVar, Generic
+from dataclasses import dataclass
+
+
+class EventType:
+    STUDY_CREATED = "STUDY_CREATED"
+    STUDY_DELETED = "STUDY_DELETED"
+    STUDY_EDITED = "STUDY_EDITED"
+    STUDY_JOB_STARTED = "STUDY_JOB_STARTED"
+    STUDY_JOB_COMPLETED = "STUDY_JOB_COMPLETED"
+
+
+@dataclass
+class Event:
+    type: str
+    payload: Any
+
+
+class IEventBus(ABC):
```

Comment:
Do we really need event bus ? Currently it takes 1 000 line of code + 2 technos (websocket, redis) to update job status. Does a simple polling is not enough ?